core/drivers/crypto/crypto_api/acipher/rsassa.c

```diff
@@ -22,6 +22,23 @@
 
 #include "local.h"
 
+static struct drvcrypt_oid *rsassa_get_hash_oid_alg(uint32_t algo)
```

Comment:
How about moving this function to `core/drivers/crypto/crypto_api/oid/hash_oid.c`?
Then it could be implemented as:
``` C
uint32_t main_alg = TEE_ALG_GET_MAIN_ALG(algo);

if (main_alg < ARRAY_SIZE(drvcrypt_hash_oid))
     return &drvcrypt_hash_oid[main_alg];

return NULL;
```

Reply:
Sure!